Assets/HoloToolkit/Utilities/Scripts/WorldAnchorManager.cs

```diff
@@ -170,6 +170,17 @@ private void Anchor_OnTrackingChanged(WorldAnchor anchor, bool located)
 
         #endregion // Event Callbacks
 #endif
+        /// <summary>
+        /// Generates the name for the anchor.
+        /// If no anchor name was specified, the name of the anchor will be the same as the GameObject's name.
+        /// </summary>
+        /// <param name="gameObjectToAnchor">The GameObject to attach the anchor to.</param>
+        /// <param name="proposedAnchorname">Name of the anchor. If none provided, the name of the GameObject will be used.</param>
+        /// <returns>The name of the newly attached anchor.</returns>
```

Comment:
The anchor is never attached in this method, so I don't think this comment is quite right,